File: 5-Problem-A-Day/168. Excel Sheet Column Title.java
class Solution {
    public String convertToTitle(int n) {
        StringBuilder ans = new StringBuilder();
        while (n > 0) {
            int last = n % 26 == 0 ? 26 : n % 26;
            ans.append(Character.toString((char) (64 + last)));
            n = (n - last) / 26;
        }
        ans.reverse();
        return ans.toString();
    }
}

File: 2021/168. Excel Sheet Column Title.cpp
class Solution {
public:
    string convertToTitle(int columnNumber) {
        string res;
        while (columnNumber > 26) {
            res += (columnNumber - 1) % 26 + 'A';    
            columnNumber = (columnNumber - 1) / 26;
        }
        res += (columnNumber - 1) % 26 + 'A';
        reverse(res.begin(), res.end());
        return res;
    }
};